openarm_hardware/include/openarm_hardware/canbus.hpp

```diff
@@ -1,6 +1,8 @@
-#ifndef CANBUS_H
-#define CANBUS_H
+// canbus.h
+#ifndef OPENARM_CANBUS_H_
+#define OPENARM_CANBUS_H_
```

Comment:
We can use `#pragma once` instead of `#ifndef`:

```suggestion
#pragma once
```